models/metafields/shopify__product_image_metafields.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(enabled=var('shopify_using_metafield', True) and (var('shopify_using_all_metafields', False) or var('shopify_using_product_image_metafields', False)) ) }}
 
-{{ get_metafields( 
-    source_object = "stg_shopify__product_image", 
-    reference_value = 'image') 
-}}
\ No newline at end of file
+{{ shopify.get_metafields(
+    source_object = "stg_shopify__product_image",
+    reference_values = ['product_image', 'image', 'productimage']
```

Comment:
It seems like here `product_image` was placed to serve as the id field. If that's the case we should remove and let the soon to be created id field parameter address that gap

Reply:
Yep! Updated.